build/sync.ps1

```diff
@@ -1,108 +1,117 @@
-﻿$choice = $args[0]
-
-function print_usage()
+﻿try

```

Comment:
Are the `try` wrappers necessary? Why not do something similar to the shell scripts?

Reply:
Yes. Otherwise the script wont return to the previous directory in case of an exception.

Reply:
Yes, but you could spawn a new powershell instance to re-run the script in the correct directory. I personally think this is a much simpler and cleaner solution since it doesn't require as much of a drastic change.

Reply:
It seems that you need the try/finally, otherwise a crash wont return you to the directory you where. I replicated the same behavior as with the shell script.